doc/en/user/source/extensions/css/index.rst

```diff
@@ -18,3 +18,4 @@ The CSS extension adds an alternative style editor to GeoServer that uses a CSS-
    styledmarks
    cookbook/index
    examples/index
+   workshop/index
```

Comment:
Tutorial is at the top of the TOC and workshop is at the bottom. What's the difference? This should be clarified.

Perhaps add a "for more details" at the end of the tutorial for now. And maybe move the tutorial to be right before the workshop.